packages/opentelemetry-semantic-conventions-ai/opentelemetry/semconv_ai/__init__.py

```diff
@@ -3,6 +3,48 @@
 SUPPRESS_LANGUAGE_MODEL_INSTRUMENTATION_KEY = "suppress_language_model_instrumentation"
 
 
+class LLMVendor(Enum):
```

Comment:
Should be `System` / `GenAISystem` by the semconv: https://opentelemetry.io/docs/specs/semconv/registry/attributes/gen-ai/#gen-ai-system